lms/static/sass/discussion/views/_thread.scss

```diff
@@ -39,11 +39,13 @@ body.discussion, .discussion-module {
     .response-header-content {
       display: inline-block;
       vertical-align: top;
-      width: flex-grid(9,12);
+      width: flex-grid(11,12);
     }
 
     .response-header-actions {
-      width: flex-grid(3,12);
+      position: absolute;
+      right: ($baseline);
```

Comment:
I think you need to do `@include right($baseline);` to account for RTL.  See https://github.com/edx/edx-platform/wiki/RTL-UI-Best-Practices

You can test by appending `?preview-lang=ar` to the end of the URL you are viewing.

Append `?clear-lang` to reset your language, or log out and log back in to clear the token from your session.